src/System.Security.Cryptography.Csp/tests/RSACryptoServiceProviderTests.cs

```diff
@@ -270,6 +277,62 @@ public static void NonExportable_Persisted()
             }
         }
 
+        [Fact]
+        public static void ImportParameters_ExponentTooBig_Throws()
+        {
+            using (var rsa = new RSACryptoServiceProvider())
+            {
+                // Verify that Unix shims and Windows Csp both throws the same exception when large Exponent imported
+                Assert.ThrowsAny<CryptographicException>(() => rsa.ImportParameters(TestData.RsaBigExponentParams));
```

Comment:
Are both of these CryptographicException with a message (and not an HR)?  if so, that could be `Throws` instead of `ThrowsAny`.  But if it's an NTE_BAD_DATA thing ThrowsAny is appropriate.